Sources/StreamChatUI/UIConfig+MessageComposer.swift

```diff
@@ -12,14 +12,29 @@ public extension _UIConfig {
             _ChatMessageComposerView<ExtraData>.self
         public var messageInputView: _ChatMessageComposerInputContainerView<ExtraData>
             .Type = _ChatMessageComposerInputContainerView<ExtraData>.self
+
         public var documentAttachmentView: _ChatMessageComposerDocumentAttachmentView<ExtraData>.Type =
             _ChatMessageComposerDocumentAttachmentView<ExtraData>.self
         public var documentAttachmentsFlowLayout: ChatMessageComposerDocumentAttachmentsCollectionViewLayout.Type =
             ChatMessageComposerDocumentAttachmentsCollectionViewLayout.self
-        public var imageAttachmentsView: _ChatMessageComposerImageAttachmentsView<ExtraData>.Type =
-            _ChatMessageComposerImageAttachmentsView<ExtraData>.self
         public var documentAttachmentsView: _ChatMessageComposerDocumentAttachmentsView<ExtraData>.Type =
             _ChatMessageComposerDocumentAttachmentsView<ExtraData>.self
+
+        /// A view that displays a collection of image attachments
+        public var imageAttachmentsView: _ChatMessageComposerImageAttachmentsView<ExtraData>.Type =
+            _ChatMessageComposerImageAttachmentsView<ExtraData>.self
+        /// The collection view of image attachments.
+        public var imageAttachmentsCollectionView: UICollectionView.Type = UICollectionView.self
+        /// The collection view layout of the image attachments collection view.
+        public var imageAttachmentsCollectionViewLayout: UICollectionViewFlowLayout.Type =
+            UICollectionViewFlowLayout.self
+        /// A view that displays the image attachment.
+        public var imageAttachmentCellView: _ChatMessageComposerImageAttachmentView<ExtraData>.Type =
+            _ChatMessageComposerImageAttachmentView<ExtraData>.self
+        /// The view cell that displays the image attachment.
+        public var imageAttachmentCollectionViewCell: _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>.Type =
+            _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>.self
+
```

Comment:
Shouldn't some of these views be at the root and some shouldn't be here at all?

Reply:
Yes, as I said in Slack, I prefer to do these restructures when auditing the ChatMessageComposerVC since I will have a better overview of how the composer works, and so will make better decisions on the refactorings 🙂